CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Fixed an issue where **validate** falsely failed with error `DS108` on descriptions ending with new lines followed by square/curly brackets.
 * Fixed an issue where **graph** commands would not clean their temporary files properly, causing successive commands to fail.
 * Fixed an issue where an error log message changed the terminal color.
+* Enhanced configurability by introducing more dynamic settings.
```

Comment:
```suggestion
* Added the <env_var1>, <env_var2>... environment variables.
```
we should document all those new env vars,
but since this PR is urgent, you can do it in a separate PR.